cmd/common/sync.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"github.com/spf13/cobra"
+
+	grpcruntime "github.com/inspektor-gadget/inspektor-gadget/pkg/runtime/grpc"
+)
+
+func AddSyncCommand(rootCmd *cobra.Command, runtime *grpcruntime.Runtime) {
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "sync",
+		Short: "Synchronize gadget information with your cluster",
```

Comment:
cluster doesn't apply to gadgetctl. Perhaps "Synchronize gadget information with remote" or "Synchronize gadget information with ig server"?